src/recipe.rst

```diff
@@ -32,12 +32,13 @@ In case you are building your first recipe using conda-forge, a step-by-step ins
 Step-by-step Instructions
 -------------------------
 
-1. Fork `example recipes <https://github.com/conda-forge/staged-recipes/tree/master/recipes>`_
-2. Within your forked copy, generate a new folder in the recipes subdirectory and copy the `meta.yml <https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml>`_ file from the example directory. Please leave the example directory unchanged!
-3. Edit the copied recipe (meta.yml) as needed.
-4. Generate the sha-256 key as described in the example recipe using the ``openssl`` tool. As an alternative you can also go to the package description on `PyPi <https://pypi.org>`_ from which you can directly copy the sha-256.
-5. Be sure to fill in the ``tests`` section. The simplest test will simply test that the module can be imported, as described in the example.
-6. Remove all irrelevant comments in the ``meta.yaml``  file.
+1. Fork the `example recipes <https://github.com/conda-forge/staged-recipes/tree/master/recipes>`_ repository.
+2. Create a new branch from the staged-recipes ``master`` branch.
+3. Within your forked copy, generate a new folder in the recipes subdirectory and copy the `meta.yml <https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml>`_ file from the example directory. Please leave the example directory unchanged!
+4. Edit the copied recipe (meta.yml) as needed.
+5. Generate the sha-256 key as described in the example recipe using the ``openssl`` tool. As an alternative you can also go to the package description on `PyPi <https://pypi.org>`_ from which you can directly copy the sha-256.
```

Comment:
It is usually spelled SHA256

Reply:
Lines should be limited to 80 chars.

Reply:
Also using the sha256sum command is much easier than opens in my opinion.

Reply:
Thank you for the review @scopatz, I'll update accordingly 

> Also using the sha256sum command is much easier than opens in my opinion.

Agreed re: `openssl`, but I think that can be a separate PR to update the recommended way to get the SHA256 key.  It will also need to be updated in the example `meta.yml` in `staged-recipes`, which currently recommends using `openssl` 

https://github.com/conda-forge/staged-recipes/blob/73d7fe0d2695a605ef4adb3f390c77ea1f07806d/recipes/example/meta.yaml#L18-L22